app/workers/moderator/sink_articles_worker.rb

```diff
@@ -0,0 +1,20 @@
+module Moderator
+  class SinkArticlesWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority, retry: 10
+
+    def perform(user_id)
+      user = User.find_by(id: user_id)
+      return unless user
+
+      articles = Article.where(user: user)
+      reactions = Reaction.where(reactable: articles)
+      new_score = reactions.sum(:points) + Reaction.where(reactable: user).sum(:points)
+      articles.update_all(score: new_score)
+    rescue StandardError => e
+      DataDogStatsClient.count("moderators.sink", 1, tags: ["action:failed", "user_id:#{user.id}"])
+      Honeybadger.notify("Error while sinking user: #{e}")
```

Comment:
Here if you just do `Honeybadger.notify(e)` it will take care of parsing the error like class and message and all that stuff. If you send it a string it logs a `RuntimeError` which can be hard to then assess what happened.

Reply:
Ohh great! Very good to know 👍